### PR TITLE
feature: add PromotionPage base markup

### PR DESCRIPTION
**Pull request by vnko1**

### CODE REVIEW

File: frontend/src/pages/PromotionsPage/PromotionsPage.module.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      margin-bottom: 38px;
+    }
+
+    @include sm {
+      display: none;
+    }
+  }
+}
+
+.promotion-cards {
```
Comment (annaBalabak) [inline frontend/src/pages/PromotionsPage/PromotionsPage.module.scss:35]: check the design, there are also hover properties for the cards

**Review by annaBalabak — CHANGES_REQUESTED** [review]

File: frontend/src/App.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ import Order from './components/Order/Order';
 import RegisterConfirm from './components/login/RegisterConfirm';
 /* eslint-disable max-len */
 import ResetPasswordPage from './components/login/ResetPassword/ResetPasswordPage';
+import PromotionsPage from './pages/PromotionsPage/PromotionsPage';
```
[the remark this answers is not in the document]
Comment (vnko1) [reply frontend/src/App.js:28]: fixed

File: frontend/src/pages/PromotionsPage/components/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export { default as PromotionCard } from './PromotionCard/PromotionCard';
```
[the remark this answers is not in the document]
Comment (vnko1) [reply frontend/src/pages/PromotionsPage/components/index.js:1]: fixed

File: frontend/src/pages/PromotionsPage/PromotionsPage.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { useScroll } from '../../hooks';
+import promotions from '../../data/promotions.json';
+import Path from '../CardPage/components/Path';
+import PromotionCard from './components/PromotionCard/PromotionCard';
```
[the remark this answers is not in the document]
Comment (vnko1) [reply frontend/src/pages/PromotionsPage/PromotionsPage.jsx:6]: fixed

File: frontend/src/pages/PromotionsPage/PromotionsPage.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    top: 0,
+    left: 0,
+    behavior: 'smooth',
+    scrollOnMount: true,
+  });
+
+  return (
+    <>
+      <Path currentPageData={{ name: 'Акції' }} currentPageType={'section'} />
+      <section className={styles['section']}>
```
[the remark this answers is not in the document]
Comment (vnko1) [reply frontend/src/pages/PromotionsPage/PromotionsPage.jsx:20]: fixed

File: frontend/src/pages/PromotionsPage/components/PromotionCard/PromotionCard.jsx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  useEffect(() => {
+    const t = setInterval(() => {
+      setTargetDay(getRemainingDays(promotionDuration));
+    }, 1000 * 60);
+
+    return () => clearInterval(t);
+  }, [promotionDuration]);
+
+  return (
+    <div className={styles['promotion']}>
```
[the remark this answers is not in the document]
Comment (vnko1) [reply frontend/src/pages/PromotionsPage/components/PromotionCard/PromotionCard.jsx:29]: fixed

File: frontend/src/pages/PromotionsPage/components/PromotionCard/PromotionCard.jsx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+      <img
+        alt={text}
+        src={imageUrl}
+        className={styles['promotion-banner']}
+        width="690"
```
[the remark this answers is not in the document]
Comment (vnko1) [reply frontend/src/pages/PromotionsPage/components/PromotionCard/PromotionCard.jsx:34]: fixed

File: frontend/src/pages/PromotionsPage/PromotionsPage.module.scss (hunk 1 above)
Comment (vnko1) [reply frontend/src/pages/PromotionsPage/PromotionsPage.module.scss]: fixed

**Review by annaBalabak — APPROVED** [review]

**Pull request merged**